python/pyspark/ml/tuning.py

```diff
@@ -141,6 +144,71 @@ def getEvaluator(self):
         """
         return self.getOrDefault(self.evaluator)
 
+    def _transfer_param_map_to_java_impl(self, pyParamMap, java_obj):
+        """
+        Transfer a Python ParamMap to a Java ParamMap which belongs to an Java estimator of
+        ValidatorParams.
+        This utility method helps CrossValidator and TrainValidationSplit implementing their
+        _transfer_param_map_to_java().
+        """
+        estimator, epms, evaluator, seed = self._to_java_impl()
```

Comment:
This is getting the Python Params from the class, not from pyParamMap.  It should get them from pyParamMap.